pkg/sfu/receiver.go

```diff
@@ -62,6 +63,9 @@ type TrackReceiver interface {
 	GetRedReceiver() TrackReceiver
 
 	GetTemporalLayerFpsForSpatial(layer int32) []float32
+
+	GetRTCPSenderReportData(layer int32) *buffer.RTCPSenderReportData
+	GetReferenceLayerRTPTimestamp(ts uint32, layer int32, referenceLayer int32) (uint32, error)
```

Comment:
Two methods in `TrackReceiver` which is used by forwarder/down tracks. Need to implement these on relay tracks also.